pkg/display/display.go

```diff
@@ -0,0 +1,70 @@
+package display
+
+import (
+	"bytes"
+	"io"
+	"time"
+
+	"github.com/gosuri/uilive"
+)
+
+type Displayer interface {
+	Display(io.Writer) bool
+}
+
+type Display struct {
+	live     *uilive.Writer
+	interval time.Duration
+	updater  Displayer
+	buffer   *bytes.Buffer
```

Comment:
Maybe make this `bytes.Buffer` so you don't have to initialize it?

Reply:
Needs to be a pointer otherwise cannot be passed an io.Reader into io.Copy.